app/client/api/authApi.js

```diff
@@ -54,7 +54,7 @@ export const registerAPI = ({body}) => {
     body: JSON.stringify(body)
     })
     .then(response => {
-      return handleResponse(response)
+      return response.json();
```

Comment:
What was the issue that handle response was causing that requires non standard handling?

Reply:
TFA returns a 401 that causes `handleResponse()` to raise an error. This is the same for the login API (where TFA is also active). Just a special case :)

https://github.com/teamsempo/SempoBlockchain/blob/713de7fffc712b9a06052002bc70434f5fe7b542/app/client/utils.js#L54-L59

Reply:
Ohhh yeah that makes sense